src/documentation/markdown/2023/lar-formatting.md

```diff
@@ -1,137 +1,165 @@
-# HMDA LAR Formatting Tool - Instructions
+# HMDA's LAR Formatting Tool
 
-1. Download the HMDA **LAR Formatting Tool for data collected in or after 2018** by selecting the link located on the Loan/Application Register (LAR) Formatting Tool webpage located at https://ffiec.cfpb.gov/tools/lar-formatting. 
+## Interaction Guide
 
-2. **Open** the HMDA LAR Formatting Tool you downloaded; If you are using Microsoft Excel 2002 or 2003, please refer to the instructions at the end of this page on how to enable macros. 
-![](https://raw.githubusercontent.com/cfpb/hmda-frontend/master/src/documentation/markdown/images/larft/Resources_page.png)
+### File Actions
 
-3. If the message “This file originated from an Internet location and might be unsafe” appears, select the **“Enable Editing”** button.
-![](https://raw.githubusercontent.com/cfpb/hmda-frontend/master/src/documentation/markdown/images/larft/Enable_Editing.png)
+Import or export your LAR data.
 
-4. If the message “Macros have been disabled” appears, select the **“Enable Content”** button.
-![](https://raw.githubusercontent.com/cfpb/hmda-frontend/master/src/documentation/markdown/images/larft/Macros_enabled.png)
+- **Upload**: Work with an existing pipe-delimited LAR file.
+- **Download**: Create a [LAR file](#lar-file) based on the currently saved LAR data.
```

Comment:
The`[LAR file](#lar-file)` isn't link to anything in the UI.

Reply:
Updated